### PR TITLE
Jpk/user dropdown

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

**Comment by jpknoll** [comment]
I still need to add a few other pages

**Comment by jSylvestre** [comment]
The failure will be fixed in my PR.

File: src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml [hunk 1]
```diff
@@ -70,17 +72,27 @@
...
+    }
+
+    @if (User.Identity.IsAuthenticated && user != null)
+    {
+        <form asp-action="logout" asp-controller="account" method="post">
+            <button class="nav-logout btn-plain row flex-center">
                 <div class="svgwrapper">
                     @Html.InlineSvg("/wwwroot/media/navicons/logout.svg")
                 </div>
+                <span class="navitem">Logout</span>
```
Comment (jSylvestre) [inline src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml]: @jpknoll Delete this one (Merge conflict).

File: src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml [hunk 2]
```diff
@@ -47,40 +48,46 @@
             </div>
             <span class="navitem">Support</span>
         </div>
+    </a>    
+
+    <a class="nav-settings" asp-controller="Teams" asp-action="Details" asp-team="@teamSlug">
```
Comment (jSylvestre) [inline src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml]: @jpknoll should be asp-route-team= here and in the sub menu choices

But, if they don't select a team, they get errors. As an admin may not be in any teams maybe put this here:
asp-route-team="@(string.IsNullOrWhiteSpace(teamSlug) ? "----" : teamSlug)

Comment (jpknoll) [reply src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml]: At some point they'll be required to select a team at login or a default one in their user settings.

Comment (jSylvestre) [reply src/Payments.Mvc/Views/Shared/_LeftNavPartial.cshtml]: An admin that sets up teams, may never be in a team...

File: src/Payments.Mvc/Controllers/FinancialAccountsController.cs [hunk 3]
```diff
@@ -50,7 +53,7 @@ public FinancialAccountsController(ApplicationDbContext context, IFinancialServi
             }
 
 
-            if (team.Accounts.Where(a => a.IsActive && a.IsDefault).Count() != 1)
+            if (!team.Accounts.Any(a => a.IsActive && a.IsDefault))
```
Comment (jSylvestre) [inline src/Payments.Mvc/Controllers/FinancialAccountsController.cs]: @jpknoll I had it this way to warn them if they had more that one default account selected.

Comment (jpknoll) [reply src/Payments.Mvc/Controllers/FinancialAccountsController.cs]: Okay. I think that should be different error messages though. I'll break it into two different if statements.

**Pull request merged**